.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug
+about: This is a bug being reported
+title: "Bug Name"
+labels: 'bug'
+assignees: ''
+
+---
+<!--- Provide a general summary of the issue in the Title above -->
+
+### Expected Behavior
+<!--- Tell us what should happen -->
+
+### Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+### Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+### Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+
+1.
+2.
+3.
```

Comment:
I don't think this is needed.

Reply:
It's there as an example to indicate the user should be listing the steps causing their bug